odk1-src/form-question-types.rst

```diff
@@ -766,102 +766,63 @@ This is especially helpful for questions with a large number of choices.
   opt_abcd,d,D
 
 
+.. _select-columns-pack-widget:
 
-.. _select-image-widget:
-
-Select widget with image choices
-""""""""""""""""""""""""""""""""""
+Select widget with columns-pack appearance
+""""""""""""""""""""""""""""""""""""""""""""""
 
 type
   :tc:`select_one {list_name}`
 appearance
-  *none*
+  *columns-pack*
 
-Select widgets support image choices.
-The images are referenced in the **choices** sheet,
-and the image files
-need to be included in the :file:`media` folder.
+When the :tc:`columns-pack` appearance is added, the app tries to accommodate as many choices in a single line as possible.
```

Comment:
Add: if the choice labels have different lengths, they will not be in even columns.

Reply:
Fixed.